backend/createStaffAndRooms.py

```diff
@@ -80,7 +80,10 @@ def create_rooms_main(rooms_file, is_hardrock=False, force_roombaht=False):
                         number=elem['Room']
                         )
 
-            features = elem['Room Features (Accessibility, Lakeview, Smoking)'].lower()
+            try:
```

Comment:
I think I'm OK with this. These fields seem important from a "placement" perspective, but roombaht doesn't really need them for core functionality. My only concern is that, even with the room-update functionality in [this](https://github.com/Index01/RoomBot/pull/60/) branch, we wouldn't be able to ever get these fields in. Maybe I could change it so those fields can be modified after the fact, but that seems potentially problematic as they really should be immutable.